src/PrestaShopBundle/Resources/views/Admin/Sell/Catalog/Product/FormTheme/combination.html.twig

```diff
@@ -56,6 +56,7 @@
            'search.placeholder': 'Search for attributes...'|trans({}, 'Admin.Catalog.Feature'),
            'modal.title': 'Generate combinations'|trans({}, 'Admin.Catalog.Feature'),
            'modal.close': 'Cancel'|trans({}, 'Admin.Actions'),
+           'label.apply-to-all-shops': 'Apply changes to all stores'|trans({}, 'Admin.Global'),
```

Comment:
```suggestion
           'label.apply-to-all-shops': 'Generate combinations for all stores'|trans({}, 'Admin.Global'),
```
Since the action is quite specific I think it's worth having dedicated wording no? @l-delin @MatShir what do you think?